packages/bvaughn-architecture-demo/components/console/renderers/shared.module.css

```diff
@@ -135,18 +135,41 @@
   color: var(--color-error);
 }
 
+.BadgeContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 0.5rem;
+  height: 0.5rem;
+  position: absolute;
+  top: 0.5rem;
+  left: 0.5rem;
+}
+
 .ColorBadge,
 .UnicornBadge {
-  position: relative;
-  display: inline-block;
   width: 0.5rem;
   height: 0.5rem;
-  margin: 0 0.5rem 0 -1rem;
+  box-shadow: 0 0 0 rgba(185, 185, 185, 0.4);
+  animation: pulse 1.2s 5;
 }
 .ColorBadge {
   border-radius: 0.5rem;
   background-color: var(--badge-color);
 }
+.DefaultColorBadge {
+  width: 0.3rem;
+  height: 0.3rem;
```

Comment:
I'm curious about `.3rem` vs `.25rem` ? It's fine, but I think we general have been going in increments of `.25rem` in other places, so I just wanted to ask to make sure we had a good reason to go with an unusual one here 😁

Reply:
Yeah, this one is intentionally deviating a little. The colored badges are `0.5` and `0.25` was too small for the default badge, so using a slightly bigger size here.